server/lib/src/idm/server.rs

```diff
@@ -1361,6 +1361,10 @@ impl<'a> IdmServerAuthTransaction<'a> {
             admin_error!("Failed to start auth ldap -> {:?}", e);
             e
         })?;
+        if self.qs_read.d_info.d_ldap_allow_unix_pw_bind == false {
```

Comment:
this is where the condition should be read as false, but currently is not

Reply:
Just set the logic to "what it should be", even if that causes test failures for now.